pkgs/by-name/ch/chadwm/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, xorg
+, imlib2
+, picom
+, feh
+, acpi
+, rofi
+}:
+
+stdenv.mkDerivation {
+  name = "chadwm";
+  version = "unstable-2024-02-14";
```

```suggestion
  version = "0-unstable-2024-02-14";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

very strange, the last time i added a package starting with 0.0.0 i was told to take it off.

---

this is a fairly recent change, i guess when people ask to make changes, there should be a reference to explain why